packages/search-ui/src/responseAdapter.js

```diff
@@ -0,0 +1,38 @@
+function adaptation1AdaptFacetValue(facetValue) {
```

Comment:
Since a lot of the logic in `adaptFacets` just copies the current object in an immutable way, I tried to call out the adaptations in discrete methods, for readability.